src/apps/orders/routers.py

```diff
@@ -76,14 +78,33 @@ def update_order(
     db: Session = Depends(get_db),
     request_user: User = Depends(authenticate_user),
 ) -> OrderOutputSchema:
-    return update_single_order(db, order, order_id, request_user.id)
+    order_check = get_single_order(db, order_id)
+    check_object_permission(user_id=order_check.user.id, request_user=request_user)
+    db_order = update_single_order(db, order, order_id)
+    return db_order
+
+
+@order_router.delete(
+    "/",
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+def delete_orders(
+    db: Session = Depends(get_db), request_user: User = Depends(authenticate_user)
+) -> Response:
+    check_permission(request_user)
+    delete_all_orders(db)
+    return Response(status_code=status.HTTP_204_NO_CONTENT)
```

Comment:
remove this functionality, order history should be stored in DB. Implement order statuses and their management